model/patch/patch.go

```diff
@@ -582,24 +596,33 @@ func (p *Patch) FilesChanged() []string {
 	return filenames
 }
 
-// SetActivated sets the patch to activated in the db
+// SetActivated sets the patch to activated in the DB, which effectively
```

Comment:
Seems as though it does finalize the patch, can we reword to something like "marks the patch as finalized?" We also may want to change this function name now since it does more than simply marking activated.

Reply:
Yeah, I guess a finalized patch used to be called activated, but the terminology has drifted over time. I updated this + added a doc comment to activated to say it means finalized.